src/components/DropDownButton/index.tsx

```diff
@@ -108,11 +108,13 @@ const Toggle = ({
 			<JoinedButton
 				{...props}
 				icon={
-					open ? (
-						<FontAwesomeIcon icon={faChevronUp} />
-					) : (
-						<FontAwesomeIcon icon={faChevronDown} />
-					)
+					<Flex>
```

Comment:
@JSReds Let's not modify the DropDownButton as it is unnecessary. Instead, in the filters we switch between `<icon> Views...` and `<icon>` as labels Alternatively, we can check `if (label || icon)` in the DropDown instead of checking only for the `label`.

Reply:
@sradevski that was my first try but in both cases the result will be something like this:

<img width="144" alt="Screenshot 2020-03-26 at 14 48 15" src="https://user-images.githubusercontent.com/7238159/77654328-39cdd600-6f71-11ea-8deb-9c81fdabaee1.png">

should we leave it like that and play with `viewsMenuButtonProps: {width: n}` from the Filters component?

Reply:
@JSReds that's fine, the only issue here is the large padding AFAIK, but that is themable and in the dashboard is lower than in rendition.